### DISCUSSION TITLE
feat: Can capacitor filesystem read iOS file just obsidian app?

### DISCUSSION THREAD

**Discussion by aquarius-wing** [ionic-team/capacitor#6029] [Q&A] [answered]
### Description
Just like the obsidian app can read iOS iCloud file.
And it can read/write file in "On My iPhone"

### Platform(s)

iOS

**Comment by jcesarmobile** [comment, marked as answer]
I don't know what obsidian app is, but no, Filesystem plugin only read/write on your own app sandbox storage, doesn't have access to Files app. 

You can make your app files appear in Files app doing this
https://capacitorjs.com/docs/apis/filesystem#ios
But you can't read from Files app if the files were created by another app

**Reply by aquarius-wing** [reply]
Thanks, you help me alot.

**Comment by aquarius-wing** [comment]
Ask another forther question, why the Filesystem plugin design to only read/write on your own app sandbox storage.Not like the native applications can read/create whole iCloud directory.

The obsidian app is [obsidian](https://apps.apple.com/us/app/obsidian-connected-notes/id1557175442)
